### ISSUE TITLE
KotlinGenerator produces compilation error in generated interfaces when <jpaAnnotations/> is set

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#10904]
The resulting code looks like this:

```kotlin
public interface INoConstraints : Serializable {
        @Column(name = "ID", precision = 10)
    public var id: Int?
...
```

Which produces:

> This annotation is not applicable to target 'member property without backing field or delegate'

**Comment by lukaseder** [comment]
One solution would be to generate `@get:Column` and `@get:Id`, etc. annotations instead of `@Column` and `@Id`, etc.

As shown here: https://stackoverflow.com/a/56225025/521799

**Comment by lukaseder** [comment]
We'll use this prefix syntax everywhere, for consistency reasons.

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0 and 3.14.4 (#10914)
